test/test_testing.py

```diff
@@ -7119,6 +7119,180 @@ def test_stop_then_start(self, request: pytest.FixtureRequest):
             assert info.status == pebble.CheckStatus.UP
             assert info.change_id, 'Change ID should not be None or the empty string'
 
+    @pytest.mark.parametrize(
+        'combine,new_layer_name',
+        [
+            (False, 'new-layer'),
+            (True, 'base'),
+            (
+                True,
+                'new-layer',
+            ),  # This doesn't have anything to combine with, but for completeness.
```

Comment:
The formatting here is awkward -- took me a moment to figure that this was just another tuple, not nested. What about this?
```suggestion
            # This doesn't have anything to combine with, but for completeness:
            (True, 'new-layer'),
```